docs/release_notes/9_1.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Current Release 9.1
+tags:
+  - 9.1
+  - 9.1 release
+  - Rocky 9.1
+---
+
+# Release Notes For Rocky Linux 9.1
+
+## Upgrading
+
+You can upgrade from Rocky Linux 9.0 to Rocky Linux 9.1 by simply running `sudo dnf -y upgrade`
+
+!!! note
+
+    Rocky Linux does not offer an upgrade path from any version of Rocky Linux 8. We recommend doing a fresh OS install to move to Rocky Linux 9.x.
+
+## Cloud Images
+
+As well as the normal install images, there are several images available for new and existing cloud platforms.
+
+In between Rocky Linux 9.0 and 9.1, images for Oracle Cloud Platform (OCP) were developed and released. The Oracle images join our existing set of images including GenericCloud, Amazon AWS (EC2), Container RootFS/OCI, Vagrant (Libvirt, VirtualBox, and VMWare), Google Cloud Platform, Microsoft Azure, and other CSP-maintained images.
+
+Additionally, The GenericCloud, EC2, and Azure images now have a variant which uses an LVM partition for the root filesystem, allowing systems administrators additional flexibility and options for configuring their systems.
+
+More information on the artifacts produced by the Cloud Special Interest Group, as well as information on how to get involved can be found on the [SIG/Cloud Wiki](https://sig-cloud.rocky.page/) page.
+
```

Comment:
Can we clarify this in a way, where we show what's actually published now?
Because since 9.1 now we have:
- boot (no local repo content), minimal (minimal, custom os environments only), dvd (full) for all 4 arches
- Live images for x86_64 and aarch64:
  - Workstation
  - Workstation Lite
  - MATE
  - XFCE
  - KDE is postponed
- Container images published to Docker Hub, official Docker Hub, Quay.io + to the CDN for all 4 arches:
  - Base
  - Minimal
  - UBI
- Cloud images:
  - GenericCloud Base for all 4 arches
  - GenericCloud LVM for x86_64 and aarch64
  - Cloud AWS EC2 Base + LVM for x86_64 and aarch64
  - Cloud OCP Base for x86_64 and aarch64
- Vagrant images for x86_64 and aarch64:
  - Libvirt
  - VBox
  - VMware (Workstation and Fusion)

As you see they are quite a lot and also a few which are new!

Reply:
```suggestion
## Image changes

As well as the normal install images, there are several images available for new and existing cloud and container platforms.

In between Rocky Linux 9.0 and 9.1, images for Oracle Cloud Platform (OCP) were developed and released. The Oracle images join our existing set of images including GenericCloud, Amazon AWS (EC2), Container RootFS/OCI, Google Cloud Platform, Microsoft Azure, and other CSP-maintained images.

Additionally, the GenericCloud, EC2, and Azure images now have a variant which uses an LVM partition for the root filesystem, allowing systems administrators additional flexibility and options for configuring their systems. And the GenericCloud image is also available for ppc64le and s390x now!

Also the build pipeline for the container (Base, Minimal, and UBI) and Vagrant (Libvirt, VirtualBox, and VMWare) images got overhauled. Vagrant images are now available for x86_64 and aarch64, and container images for all 4 arches.

More information on the artifacts produced by the Cloud Special Interest Group, as well as information on how to get involved can be found on the [SIG/Cloud Wiki](https://sig-cloud.rocky.page/) page.

```